tests/assets/sub0/misc/cato_structural_config.json

```diff
@@ -0,0 +1,56 @@
+{
+   "general": {
+      "fslRootDir": "/home/work/local/fsl/",
+      "freesurferRootDir": "/usr/local/freesurfer/7.2.0",
```

Comment:
To be removed/updated depending on how we handle these dependencies

Reply:
It is indeed desirable for the test framework to be independent of FSL and FreeSurfer. For as long as the test framework depends on these tools, it might be easiest to define them at one place, such that they can easily be adjust when someone runs the tests. As a temporary solution, I would suggest to provide fslRootDir and freesurferRootDir as input variables to structural_pipeline in phantomTest.m.

Reply:
fixed in https://github.com/dutchconnectomelab/CATO/pull/3/commits/6db5d6c48e75ae566754e9e671d83997a6c24317